lgn-worker/src/manager/v1.rs

```diff
@@ -18,41 +17,35 @@ pub(crate) fn register_v1_provers(
         .instance_type
         >= TaskDifficulty::Small
     {
-        debug!("Creating v1 query prover");
         register_v1_query(
             config,
             manager,
         )
         .context("failed to register the query prover")?;
-        debug!("Query prover created");
     }
 
     if config
         .worker
         .instance_type
         >= TaskDifficulty::Medium
     {
-        debug!("Creating v1 preprocessing prover");
         register_v1_preprocessor(
             config,
             manager,
         )
         .context("failed to register the pre-processing prover")?;
-        debug!("Preprocessing prover created");
     }
 
     if config
         .worker
         .instance_type
         >= TaskDifficulty::Large
     {
-        debug!("Creating groth16 prover");
         register_v1_groth16(
             config,
             manager,
         )
         .context("failed to register the groth16 prover")?;
-        debug!("Groth16 prover created");
```

Comment:
This will call the `create_prover` functions that already log the prover creation.